frontend/src/app/components/containers/complaints/outcomes/oucome-by-animal/animal-outcome.tsx

```diff
@@ -0,0 +1,282 @@
+              text="Edit"
+              icon={BsPencil}
+              click={() => hendleEnableEditMode()}
+            />
+          </div>
+        </div>
+      </div>
+      <div className="comp-details-edit-container">
+        <div className="comp-details-edit-column">
+          <div className="comp-details-edit-container comp-details-nmargin-right-xxl">
+            <div className="comp-details-edit-column">
+              <div className="comp-details-label-div-pair ">
+                <label
+                  className="comp-details-inner-content-label"
+                  htmlFor="comp-review-required-officer"
+                >
+                  Animal
+                </label>
+                <div className="flex-container">
+                  <div>
+                    <b>{animal}</b>
+                  </div>
+                  {data?.sex && (
+                    <>
+                      <div className="comp-margin-right-xs">,</div>
+                      <div>{animalSex}</div>
+                    </>
+                  )}
+                  {data?.age && (
+                    <>
+                      <div className="comp-margin-right-xs">,</div>
+                      <div className="comp-margin-right-xs">{animalAge}</div>
+                    </>
+                  )}
+                  {data?.threatLevel && (
+                    <div className="badge comp-status-badge-threat-level comp-margin-right-xs">
+                      Category level: {animalThreatLevel}
+                    </div>
+                  )}
+                  {data?.conflictHistory && (
+                    <div className="badge comp-status-badge-conflict-history comp-margin-right-xs">
+                      Conflict history: {animalHistory}
+                    </div>
+                  )}
+                </div>
+              </div>
+            </div>
+          </div>
+
+          {data?.tags && from(data?.tags).any() && (
+            <div className="comp-details-edit-column">
+              <div className="comp-details-label-input-pair">
+                <label className="comp-details-inner-content-label top">Ear Tag{data?.tags.length > 1 && "s"}</label>
+
+                <div className="comp-animal-outcome-fill-space">
+                  <ul className="comp-ear-tag-list">
+                    {data?.tags.map(({ id, identifier, ear }) => (
+                      <li key={id}>
+                        {identifier} {ear === "L" ? leftEar?.label : rightEar?.label} side
+                      </li>
+                    ))}
+                  </ul>
+                </div>
+              </div>
+            </div>
+          )}
+
+          {data?.drugs && from(data?.drugs).any() && (
+            <div className="comp-details-edit-column">
+              <div className="comp-details-label-input-pair">
+                <label className="comp-details-inner-content-label top">{`Drug${data?.drugs.length}s`}</label>
```

Comment:
I'm guessing this should be a short-circuit evaluation to determine if the s apepars or not.  Currently, the length is included in the word Drug.

Reply:
yup, something like that